index3.html

```diff
@@ -0,0 +1,104 @@
+															етнічне походження
+				</div>
+				<div class="percentage_distribution_text">Результати тесту вашої ДНК від MyHeritage включають<br>
+														визначення етнічного походження: процентний розподіл<br>
+														вашого походження, що показує походження ваших<br>
+														предків серед 42 етнічних груп, включаючи японську,<br>
+														ірландську, італійську, єврейську та багатьох інших. Ви<br>
+														знайдете список регіонів, з яких походять ваші предки<br>
+														— з обох сторін вашої родини, — а також відсотковий <br>
+														склад вашої ДНК, що бере початок із кожного регіону.
+					<img class="line1" src="assets/line.png">
+				</div>
+			</section>
+			<section class="Genetic_groups">
+				<div class="Genetic_groups_title">Отримайте детальне<br>
+												розуміння свого<br>
+												походження
+				</div>
+				<div class="Genetic_groups_text">Генетичні групи дозволяють вдосконалити вашу етнічну<br>
+												приналежність, визначаючи окремі групи, які ви<br>
+												успадкували з відповідного географічного регіону.<br>
+												Завдяки генетичним групам ви дізнаєтесь про місця, з<br>
+												яких походять ваші предки серед 2 114 груп, надаючи<br>
+												краще розуміння вашого родинного походження.
+					<img class="line2" src="assets/line.png">
+			</section>
+			<section class="regional_groups">
+				<div class="regional_groups_title">Визначає<br>
+													найрізноманітніші<br>
+													групи регіонів
+				</div>
+				<div class="regional_groups_region">Північна та Західна<br>
+													Європа
+					<img class="line3" src="assets/line2.png">
+					<img class="s_button" src="assets/sound_button.png">
+				</div>
+			</section>
+			<section class="founders_of_the_family">
+				<div class="founders_of_the_family_title">Проект засновників роду
+				</div>
+				<div class="founders_of_the_text">Компанія MyHeritage визначила унікальних засновників родів,<br>
+													людей які жили в одному місці протягом багатьох поколінь.<br>
+													Ми скерували тести ДНК до цих людей з усього світу від<br>
+													Узбекистану до Фіджі, а також від Гренландії до Південної<br>
+													Африки, які були обрані на основі їхнього походження<br>
+													генетичними представниками в їх географічному регіоні. Цей<br>
+													проект — найбільший у своєму роді з усіх, які коли-небудь<br>
+													проводилися — дозволяє MyHeritage виділяти набагато<br>
+													більше етнічних груп, ніж будь-яка інша велика служба<br>
+													тестування ДНК<br>
+													<a href="">Дізнайтеся більше</a>
+				</div>
+			</section>
+			<section class="distribution_of_your_ancestry">Отримайте розгалужений розподіл вашого походження —<br>
+															<b>2 114 регіонів</b>
+				<div class="distribution_of_your_ancestry_button">
+						<button class="distribution_of_your_ancestry_button_design">
+							<a href="">Замовити зараз</a>
```

Comment:
Why not just to style `anchor` to look like a button, so you will avoid such 'not correct' nesting `a` into `button` tag. I mean you can use just `anchor` w/out button(you can remove it at all)